src/main/java/io/confluent/idesidecar/restapi/cache/AdminClients.java

```diff
@@ -34,9 +35,15 @@ public AdminClient getClient(String connectionId, String clusterId) {
         clusterId,
         () -> {
           // Generate the Kafka admin client configuration
-          var config = configurator.getAdminClientConfig(connectionId, clusterId, false);
+          var config = configurator.getAdminClientConfig(connectionId, clusterId);
+          Log.debugf(
+              "Creating schema registry client for connection %s and cluster %s with configuration:\n  %s",
+              connectionId,
+              clusterId,
+              config
+          );
           // Create the admin client
-          return AdminClient.create(config);
+          return AdminClient.create(config.asMap());
```

Comment:
Changing the `ClientConfigurator.get*(...)` methods to return a `Configuration` object makes it much easier to get the configuration once and then use the redacted form in log message and pass the non-redacted form to the client.